api/bulk.go

```diff
@@ -0,0 +1,32 @@
+// generated by github.com/elastic/go-elasticsearch/cmd/generator; DO NOT EDIT
+
+package api
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Bulk makes it possible to perform many index/delete operations in a single API call. See https://www.elastic.co/guide/en/elasticsearch/reference/5.x/docs-bulk.html for more info.
+//
+// body: the operation definition and data (action-data pairs), separated by newlines.
+//
+// options: optional parameters. Supports the following functional options: WithIndex, WithType, WithSource, WithSourceExclude, WithSourceInclude, WithFields, WithPipeline, WithRefresh, WithRouting, WithTimeout, WithTypeParam, WithWaitForActiveShards, WithErrorTrace, WithFilterPath, WithHuman, WithPretty, WithSourceParam, see the Option type in this package for more info.
```

Comment:
Unfortunately I don't think so. It's also not trivial to hack around it since for instance "body" comes from the param object (Name field) and the description comes mostly from the spec (after some formatting/enhancing). I think the easiest to handle this is to write a script that does it for you after the fact, let me look around for one I bet there is.